s1m0nWasTaken/CS4337_BigDataProject
# Alex/add message broker

## Summary

add a rabbitmq between paymentservice and shopservice for stock changes
has a dead letter queue aswell

## How to verify

start eurika,shop,payment and rabbit. send a /payment/create request to paymentservice. open the rabbit dashboard to see it working

## Tests


## Related
